src/wallet/rpcwallet.cpp

```diff
@@ -2804,9 +2804,9 @@ UniValue bumpfee(const JSONRPCRequest& request)
             nNewFeeRate = CWallet::fallbackFee;
         }
 
-        // new fee rate must be at least old rate + minimum relay rate
-        if (nNewFeeRate.GetFeePerK() < nOldFeeRate.GetFeePerK() + ::minRelayTxFee.GetFeePerK()) {
```

Comment:
ok i don't mind doing that, but will just do for the changes introduced in this PR here?